res/card/template/css/card.css

```diff
@@ -108,7 +108,7 @@
 }
 
 .rules {
-  font-family: 'Segoe UI Semibold';
+  font-family: 'Segoe UI';
```

Comment:
Semibold seems to be throwing off the generator 🤷  Semibold is the only version I have installed, so this works. Eventually I'll figure out how to make this portable (tracked [here](https://github.com/elliottomlinson/cardmaster/issues/150))